Atomic/atomic.py

```diff
@@ -301,6 +313,15 @@ def is_iid(self):
                 return True
         return False
 
+    def get_image_id(self):
```

Comment:
We should be using the id from the image or container objects for this.  See Atomic/objects/image.py for example